lib/grammars.coffee

```diff
@@ -253,6 +253,11 @@ module.exports =
       command: "rails"
       args: (context) -> ['runner', context.filepath]
 
+  Rust:
+    "File Based":
+      command: "bash"
+      args: (context) -> ['-c', "rustc " + context.filepath + " -o /tmp/rs.out && /tmp/rs.out"]
```

Comment:
> wrapping everything as bash one-liner

Silly me. You're absolutely right in this working. This works for me for now.

> If you agree to merge it I can send another patch with fixes for those definitions:

Feel free to propose the patches, we appreciate the contributions. @erran does most of the maintenance and we could use more development effort to address efforts.